scripts/README.md

```diff
@@ -0,0 +1,90 @@
+# admin scripts
+
+**Note**: Please replace '2019-07-07' with the name of the repository.
+
+Prerequisites:
+
+- `docker` CLI available in the system
+
+when setting up a new machine for hackathons:
+
+```sh
+# run from the top of the source tree
+scripts/create_hackathon_account
+```
+
+after that, the flow of the admin steps is:
+
+## foreach hackathon:
+
+1. create a local clone of the hackathon repo in `/home/hackathon`, e.g.:
+
+```sh
+cd /home/hackathon
+git clone https://githib.com/pmemhackathon/2019-07-07
+```
+
+2. create users, e.g.:
+
+```sh
+# creates all 200 accounts
+create_pmemusers 1 200 2019-07-07
+```
+
+3. create a directory for the workshop
+
+```sh
+cd /home/hackathon/workshops
+mkdir workshop_name
+```
+
+## foreach session with same users (or on system boot)
+
+start the webhackathon daemon as root
+
+```sh
+cd /home/hackathon
+./webhackathon 2019-07-07 &
+enable_pmemusers 1 100 todayspasswd
```

Comment:
What directory should be this command run from ?

Reply:
Done.